package/linux/debian-control/postinst.in

```diff
@@ -77,7 +77,16 @@ if test "$INIT_SYSTEM" = "systemd"
 then
    systemctl stop rstudio-server.service 2>/dev/null
    systemctl disable rstudio-server.service 2>/dev/null
-   cp ${CMAKE_INSTALL_PREFIX}/extras/systemd/rstudio-server.service /lib/systemd/system/rstudio-server.service
+  
+   if test -f /lib/systemd
```

Comment:
Shouldn't this be `-e`? (since `-f` tests for existence of 'regular' file)

Reply:
You're right, thanks! Will update.